Documentation/network/kubernetes/configuration.rst

```diff
@@ -150,65 +144,59 @@ The CNI auto installation is performed as follows:
 1. The ``/etc/cni/net.d`` and ``/opt/cni/bin`` directories are mounted from the
    host filesystem into the pod where Cilium is running.
 
-2. The file ``/etc/cni/net.d/05-cilium.conf`` is written in case it does not
-   exist yet.
-
-3. The binary ``cilium-cni`` is installed to ``/opt/cni/bin``. Any existing
+2. The binary ``cilium-cni`` is installed to ``/opt/cni/bin``. Any existing
    binary with the name ``cilium-cni`` is overwritten.
 
-.. _install_cni:
-
-Manually installing CNI
------------------------
-
-This step is typically already included in all Kubernetes distributions or
-Kubernetes installers but can be performed manually:
-
-.. code-block:: shell-session
-
-    sudo mkdir -p /opt/cni
-    wget https://storage.googleapis.com/kubernetes-release/network-plugins/cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz
-    sudo tar -xvf cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz -C /opt/cni
-    rm cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz
+3. The file ``/etc/cni/net.d/05-cilium.conflist`` is written.
 
 
 Adjusting CNI configuration
 ---------------------------
 
-The CNI configuration file is automatically written and maintained by the
-scripts ``cni-install.sh`` and ``cni-uninstall.sh`` which are running as
-``postStart`` and ``preStop`` hooks of the Cilium pod.
```

Comment:
Does removing this has any impact on https://github.com/cilium/cilium/pull/24375?

Reply:
Oh, good question! I took a look and: no, we're good. That PR allows for tweaking the nodeinit daemonset, not the preStart hook for the agent itself.